lib/pbench/test/unit/server/conftest.py

```diff
@@ -2,8 +2,7 @@
 import tempfile
 import pytest
 from pathlib import Path
-
-from pbench.server.api import create_app
+from pbench.server.api import create_app, get_server_config
```

Comment:
Not something to change here, but PEP 8 specifies that the groups of the different "kinds" of imports (sorted) are separated by a space.  But since `black` and `flake8` don't really catch that we can live with it.

So really the imports should look like:

```
import pytest
import tempfile

from pathlib import Path

from pbench.server.api import create_app, get_server_config
```

Reply:
I see, got it